src/connection.cpp

```diff
@@ -68,12 +68,16 @@ Connection::Init(Napi::Env env, Napi::Object exports)
 Connection::Connection(const Napi::CallbackInfo& info)
   : Napi::ObjectWrap<Connection>(info)
 {
+  // We cannot create the Instance here b/c we need the timeout config.
+  // The instance will be created in jsConnect()
 }
 
 Connection::~Connection()
 {
-  _instance->asyncDestroy();
-  _instance = nullptr;
+  if (_instance != nullptr) {
```

Comment:
Why is this being changed in a library update?

Reply:
Fair point.  This change helped testing move along.  Given the state of Columnar, I opted for grouping changes that were all related in order to move quicker.